ogre/src/OgreLidarVisual.cc

```diff
@@ -305,11 +328,14 @@ void OgreLidarVisual::Update()
                               new OgreDynamicLines(MT_POINTS));
 
         #if (OGRE_VERSION <= ((1 << 16) | (10 << 8) | 7))
-              line->setMaterial("Lidar/BlueRay");
+              Ogre::MaterialPtr mat =
+                  Ogre::MaterialManager::getSingleton().getByName(
+                                                    "PointCloudPoint");
+            line->setMaterial(mat->getName());
         #else
             Ogre::MaterialPtr mat =
                   Ogre::MaterialManager::getSingleton().getByName(
-                                                    "Lidar/BlueRay");
+                                                    "PointCloudPoint");
```

Comment:
optimization suggestion: can you move the `Ogre::MaterialManager::getSingleton().getByName` calls outside the for loop so we don't have to search for the material every iteration of the loop.

same for the other materials like `Lidar/BlueStrips`, `Lidar/TransBlack`, `Lidar/LightBlueStrips`